dashboard/src/api.js

```diff
@@ -199,7 +163,7 @@ export async function getLocationData(location) {
 
 export async function getCoords(cityName) {
   try {
-    const url = `https://api.positionstack.com/v1/forward?access_key=${AN_API_KEY}&query=${encodeURIComponent(cityName)}`;
+    const url = `http://api.positionstack.com/v1/forward?access_key=${AN_API_KEY}&query=${encodeURIComponent(cityName)}`;
```

Comment:
I'm guessing this is needed to be able to call the API?

Reply:
Https is a premium service under this API :D